automatic/vlc.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,21 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsDir = Split-Path $MyInvocation.MyCommand.Definition
+
+#Remove old versions
+Get-ChildItem -Path $toolsDir | Where-Object { $_.PSIsContainer } | Remove-Item -EA 0 | Out-Null
```

Comment:
This will not work if VLC is in use, so it should kill it beforehand.

Reply:
Fixed.